openthread/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(CONFIG_OPENTHREAD_SOURCES)
   set(OPENTHREAD_UTILS_DST_DIR "${NRFXLIB_DIR}/openthread/include/utils")
   set(OPENTHREAD_SYSTEM_H "${ZEPHYR_OPENTHREAD_MODULE_DIR}/examples/platforms/openthread-system.h")
   set(OPENTHREAD_UART_H "${ZEPHYR_OPENTHREAD_MODULE_DIR}/examples/platforms/utils/uart.h")
-  set(NRF_SECURITY_MBEDTLS_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/../nrf_security/src/include/generated/${CONFIG_MBEDTLS_CFG_FILE})
+  set(NRF_SECURITY_MBEDTLS_CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/../../../nrf/subsys/nrf_security/src/include/generated/${CONFIG_MBEDTLS_CFG_FILE})
```

Comment:
Why can't this refer to the nrf module variable instead?

Reply:
which one ?

Reply:
`${ZEPHYR_NRF_MODULE_DIR}`

Reply:
This unwinds on my env to `$HOME/Repos/ncs/nrf/` and I want to copy build generated mbedtls config file.

Reply:
Oh it's a generated file, will approve, my bad